src/Todo.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+function Todo({ todo, onClickDelete }) {
+  return (
+    <div>
+      {todo.title}
+      <button type="button" onClick={() => onClickDelete(todo)}>
```

Comment:
```suggestion
      <button type="button" onClick={() => onClickDelete(id)}>
```

todo를 그럼 구조분해 할당 할 수 있겠죠?

Reply:
제가 이해한 바가 맞는지 정확히 모르겠지만, 수정 후 검토 받겠습니다!! ㅎㅎㅎ